cmd/src/snapshot_upload.go

```diff
@@ -113,7 +113,7 @@ BUCKET
 			progress := out.Progress(progressBars, nil)
 			progress.WriteLine(output.Emoji(output.EmojiHourglass, "Starting uploads..."))
 			bucket := c.Bucket(*bucketName)
-			g := group.New().WithErrors().WithContext(ctx)
+			g := pool.New().WithErrors().WithContext(ctx)
```

Comment:
Shiny!

Reply:
I had to, the `lib/group` package was deleted :lolsob: 😬